autopilot/sagemaker_autopilot_abalone_parquet_input.ipynb

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    from sklearn.datasets import load_svmlight_file
</pre><p>Looks like this import is not used and can be deleted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3093/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done, thank you</p>